src/scenes/home/home-drawer.component.tsx

```diff
@@ -1,85 +1,89 @@
-import React from 'react';
+import React, { ReactElement, useState } from 'react';
 import { StyleSheet, View } from 'react-native';
 import {
   Avatar,
   Divider,
   Drawer,
+  DrawerItem,
   DrawerElement,
-  DrawerHeaderElement,
-  DrawerHeaderFooter,
-  DrawerHeaderFooterElement,
   Layout,
-  MenuItemType,
   Text,
+  IndexPath,
 } from '@ui-kitten/components';
 import { BookIcon, GithubIcon } from '../../components/icons';
 import { SafeAreaLayout } from '../../components/safe-area-layout.component';
 import { WebBrowserService } from '../../services/web-browser.service';
 import { AppInfoService } from '../../services/app-info.service';
 
-const DATA: MenuItemType[] = [
-  { title: 'Libraries', icon: GithubIcon },
-  { title: 'Documentation', icon: BookIcon },
-];
-
 const version: string = AppInfoService.getVersion();
 
 export const HomeDrawer = ({ navigation }): DrawerElement => {
+  const [selectedIndex, setSelectedIndex] = useState<IndexPath>(null);
 
-  const onItemSelect = (index: number): void => {
-    switch (index) {
-      case 0: {
+  const DATA = [
+    {
+      title: 'Libraries',
+      icon: GithubIcon,
+      onPress: () => {
         navigation.toggleDrawer();
         navigation.navigate('Libraries');
-        return;
-      }
-      case 1: {
-        WebBrowserService.openBrowserAsync('https://akveo.github.io/react-native-ui-kitten');
+      },
+    },
+    {
+      title: 'Documentation',
+      icon: BookIcon,
+      onPress: () => {
+        WebBrowserService.openBrowserAsync(
+          'https://akveo.github.io/react-native-ui-kitten'
+        );
```

Comment:
Why not in the same line if fits?
```suggestion
      onPress: () => {
        WebBrowserService.openBrowserAsync('https://akveo.github.io/react-native-ui-kitten');
```